src/wattsi.pas

```diff
@@ -1319,15 +1298,6 @@    TCrossReferences = record
          end
          else
          begin
-            if (Variant <> vDEV) then
```

Comment:
Wait, we still need this for caniuse warnings! Those are actionable :)

Reply:
>```
>-           if (Variant <> vDEV) then
>```
>
> Wait, we still need this for caniuse warnings! Those are actionable :)

We still get the caniuse warnings, regardless. That conditional block was just creating a separate code path that handled reporting of both the bugzilla annotations and caniuse warnings together. The `else` for that conditional block repeats the exact same handling for just caniuse case. Here’s the full context:

```
-            if (Variant <> vDEV) then
-            begin
-               Warn('Could not find ID ' + ID + ' for annotation that uses URLs:');
-               for Bug in Feature.Bugs do
-                  Writeln('   ', Bug.URL);
-               if (Feature.CanIUseCode <> '') then
-                  Writeln('   https://caniuse.com/#feat=', Feature.CanIUseCode);
-            end
-            else
             if (Feature.CanIUseCode <> '') then
             begin
                Warn('Could not find ID ' + ID + ' for annotation that uses URLs:');
                Writeln('   https://caniuse.com/#feat=', Feature.CanIUseCode);
             end;
```